Gemfile

```diff
@@ -11,7 +11,7 @@ gem "govuk_sidekiq"
 gem "inherited_resources"
 gem "kaminari-actionview"
 gem "kaminari-mongoid"
-gem "mongo", "2.15.1"
+gem "mongo", "2.18.1"
```

Comment:
You need to downgrade this - 2.18.1 drops support for Mongo 2.6, and I haven't quite got the Postgres version ready for deploy yet.